modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java

```diff
@@ -1281,7 +1281,7 @@ public Response addDCCatalog(@FormParam("mediaPackage") String mp, @FormParam("d
       logger.error("Could not write catalog to disk: {}", e.getMessage());
       return Response.serverError().build();
     } catch (Exception e) {
-      logger.error(e.getMessage());
+      logger.error("Unable to add catalog: {}", e.getMessage());
```

Comment:
Not requiring a change for this pull, I'm just noting the oddity that the failed mediapackage gets a WARN log. But, a failed add catalog is a full blown ERROR [system about to do down in a fiery mess] log.

IMHO, ERROR logs warrant a full stack trace, not just a potentially missing message.

Reply:
An IOException from filesystem full, incorrect filesystem permissions or something would be  ERROR-worthy.

Anyway, changes to log level outside the scope of this PR.